test/ufuzz.js

```diff
@@ -630,8 +630,12 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
             );
             break;
           default:
+            if (rng(4) == 0) s.push('function ' + name + '(){');
+            else {
+                VAR_NAMES.push('this');
```

Comment:
What prevents `'this'` from being pushed multiple times? And what removes it after the `new function`?